react-main/src/redux/game/actions.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
as to avoid repeating the action names, what we usually do is define them in a constant in actions.js and import them in the reducer:

```
export const actions = {
  SELECT: 'SELECT',
  HISTORY: 'HISTORY'
}

export const selectSquare = position => ({type: actions.SELECT, payload: position})

// reducer:

case actions.SELECT: ...
```